packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -415,18 +419,32 @@ const GroupForm = ({
                 />
               ))}
 
-              {requirementSubForms.length < MAX_REQUIREMENTS && (
-                <CWButton
-                  type="button"
-                  label="Add requirement"
-                  iconLeft="plus"
-                  buttonWidth="full"
-                  buttonType="secondary"
-                  buttonHeight="med"
-                  onClick={addRequirementSubForm}
-                />
-              )}
-
+              <CWTooltip
```

Comment:
i am still seeing the tooltip not showing up or not hiding sometimes. 

https://github.com/hicommonwealth/commonwealth/assets/51641047/9221c234-8786-4433-84b7-bf28a28220c5